README.rst

```diff
@@ -263,9 +263,12 @@ When running `Robot Framework`_ tests with `--xunit report.xml`_ as an input
 argument, an xUnit compatible result file is generated. The warnings-plugin can
 parse this file and check the amount of failures. By default, the test results
 of all test suites in the file are taken into account. If you only care about
-one specific test suite, you can use ``--name <<suite name>>``. The warning
+one specific test suite, you can use ``--name <<suite name>>``. If this suite
+name doesn't exist in the input file, an error is raised. The warning
```

Comment:
```suggestion
name doesn't exist in the input file, an error can be raised. The warning
```

Reply:
I assume this change has become undesired

Reply:
it indeed has.